README.md

```diff
@@ -120,6 +120,7 @@ Supported Cars
 | Toyota               | Rav4 Hybrid 2016           | TSS-P                | Yes     | Yes<sup>2</sup>| 0mph             | 0mph           | Toyota            |
 | Toyota               | Rav4 Hybrid 2017-18        | All                  | Yes     | Yes<sup>2</sup>| 0mph             | 0mph           | Toyota            |
 | Toyota               | Sienna 2018                | All                  | Yes     | Yes<sup>2</sup>| 0mph             | 0mph           | Toyota            |
+| Toyota               | Toyota Hybrid 2019         | All                  | Yes     | Yes            | 0mph             | 0mph           | Toyota            |
```

Comment:
this should be:
```
| Toyota               | Toyota Corolla Hybrid 2020      | All                  | Yes     | Yes            | 0mph             | 0mph           | Toyota            |
```
And placed in alphabetical order in the table (after Corolla Hatchback)